bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java

```diff
@@ -100,6 +100,7 @@ public class TabDebugger extends AbstractLaunchConfigurationTab
 
 	// ------------------------------------------------------------------------
 
+	private static final String LAUNCH_TARGET_NAME_ATTR = "com.espressif.idf.launch.serial.core.idfTarget"; //$NON-NLS-1$
```

Comment:
had to write this attribute like this. Maybe we can use single class in core package to store all attributes there

Reply:
You can use IDFLaunchConstants.ATTR_IDF_TARGET